Server/src/main/java/org/openas2/app/partner/AddPartnerCommand.java

```diff
@@ -1,12 +1,18 @@
 package org.openas2.app.partner;
 
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.bouncycastle.mail.smime.handlers.pkcs7_signature;
 import org.openas2.OpenAS2Exception;
 import org.openas2.cmd.CommandResult;
 import org.openas2.partner.PartnershipFactory;
 import org.openas2.partner.XMLPartnershipFactory;
 import org.w3c.dom.Document;
 import org.w3c.dom.Element;
 
+import edu.umd.cs.findbugs.log.Logger;
+import io.sentry.instrumentation.file.SentryFileInputStream.Factory;
```

Comment:
The default logger is the apache Commons logging package.
Alternative loggers must be plugged in - see the logging section in the OpenAS2HowTo.